--- a/src/libexpr/flake/flake.cc
+++ b/src/libexpr/flake/flake.cc
@@ -74,7 +74,7 @@

 static void forceTrivialValue(EvalState & state, Value & value, const PosIdx pos)
 {
-    if (value.isThunk() && value.isTrivial())
+    if (value.isThunk()) // HACK: Evaluate even if it's not trivial
         state.forceValue(value, pos);
 }

@@ -218,7 +218,7 @@
         throw Error("source tree referenced by '%s' does not contain a '%s/flake.nix' file", lockedRef, lockedRef.subdir);

     Value vInfo;
-    state.evalFile(flakeFile, vInfo, true); // FIXME: symlink attack
+    state.evalFile(flakeFile, vInfo, false); // FIXME: symlink attack

     expectType(state, nAttrs, vInfo, state.positions.add({flakeFile}, 1, 1));

+

